src/timeseriesflattenerv2/spec_processors/static.py

```diff
@@ -12,15 +12,16 @@
 def process_static_spec(
     spec: StaticSpec, predictiontime_frame: PredictionTimeFrame
 ) -> ProcessedFrame:
-    new_col_name = (
-        f"{spec.column_prefix}_{spec.value_frame.value_col_name}_fallback_{spec.fallback}"
-    )
+    new_col_names = [
```

Comment:
Turn this into `old2new_colname`? Should make the rename and select statements much easier to read.